man/meson.build

```diff
@@ -184,17 +184,20 @@ html = custom_target(
         depends : html_pages,
         command : ['echo'])
 
-run_target(
-        'doc-sync',
-        depends : man_pages + html_pages,
-        command : ['rsync', '-rlv',
-                   '--delete-excluded',
-                   '--include=man',
-                   '--include=*.html',
-                   '--exclude=*',
-                   '--omit-dir-times',
-                   meson.current_build_dir(),
-                   get_option('www-target')])
+rsync = find_program('rsync', required : 'false')
```

Comment:
actually lgtm-not:

> man/meson.build:187:0: ERROR: required keyword argument must be boolean or a feature option

shouldn't be a string

Reply:
Sorry.

Reply:
third time's the charm!